notebooks/README.md

```diff
@@ -4,9 +4,7 @@ These notebooks provide examples of how to use cuDF.  These notebooks are design
 
 ## Notebooks
 Notebook Title | Status | Description     
---- | --- | ---                                                                                                                                                                                           
-[Numba-cuDF Integration](notebooks_numba_cuDF_integration.ipynb) | Working | A demonstration of GPU accelerated Python library interoperability, including a few examples of accelerating cuDF Dataframe operations using Numba kernels directly.
-[Apply Operations in cuDF](notebooks_Apply_Operations_in_cuDF.ipynb) | Working | Accelerated, customized data transformation has been found to be very valuable. cuDF provides two special methods that serve this particular purpose: apply_rows and apply_chunks functions, which utilize the Numba library to accelerate the data transformation via GPU in parallel. This notebook shows a few examples of how to use them.
+--- | --- | ---
```

Comment:
Might just remove this entirely, since it looks to now be superseded by the section below.

Reply:
@dillon-cullinan Is it okay to remove this or do you want this kept here to be aligned with the other repos?

Reply:
Is cuDF planning on having notebooks in the future or no? If not I'd just drop it and keep the reference to the notebooks repo.

Reply:
Not sure, will remove it for now and we can always add back later